screen2.0/src/app/search/_ccredetails/queries.ts

```diff
@@ -373,4 +373,44 @@ const LINKED_GENES_CELLTYPES = gql`
       method
     }
   }
+`
+export const TSS_RAMPAGE_PEAKS = gql`
+query getTSSRampagePeaks($coordinates: ChromRange!) {
+  gettssRampagePeaks(coordinates: $coordinates) {
+    peakType
+    start
+    chrom
+    end
+    peakId
+    genes 
+    {
+      geneName      
+      locusType
+    }
+  }
+}
+`
+
+export const TSS_RAMPAGE_QUERY = gql`
+query tssRampage($peak: String!) {
```

Comment:
Can you make this query support sending either gene or peak so that Rampage.tsx can use it also (want to remove the query from that file and import this one)? Lmk if you have trouble with this